packages/generator/src/renderers/index.ts

```diff
@@ -0,0 +1,14 @@
+import type { Renderer } from '../types';
+import textRenderer from './textRenderer';
+import imageRenderer from './imageRenderer';
+import barcodeRenderer from './barcodeRenderers';
+import { BarCodeType } from '@pdfme/common';
+
+const barCodeTypes: BarCodeType[] = ['qrcode', 'japanpost', 'ean13', 'ean8', 'code39', 'code128', 'nw7', 'itf14', 'upca', 'upce', 'gs1datamatrix']
+
+const renderer: Renderer = {
+    text: { renderer: textRenderer, },
```

Comment:
ok, so this is where the different renderers are mapped 👍 

perhaps: 
``` 
   text: { render: renderText }, 
   image: { render: renderImage }.`
```
etc? 

this naming feels more functional rather than classes/objects, wdyt?

Reply:
You are right!
Since it is just a function rather than a renderer, we will use Render.

Also, I will rename textRenderer.ts and imageRenderer.ts to textRender.ts and imageRender.ts.